chunker/scripts/runBenchmarkTest.sh

```diff
@@ -17,7 +17,7 @@ for JAR in `ls $LIB`; do
 done
 
 
-TESTDATA="src/main/resources/test.txt"
+TESTDATA="/shared/corpora/corporaWeb/written/eng/chunking/conll2000distributions/test.txt"
```

Comment:
This script is running [ChunkTester](https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/blob/master/chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkTester.java), which doesn't take any file as input. I think one day someone modified the interface of ChunkTester but didn't update runBenchmarkTest.sh.